angular-ngrx-scss/src/app/home/home-routing.module.ts

```diff
@@ -2,13 +2,19 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { AuthGuard } from '../auth/services/auth.guard';
 import { HomeComponent } from './home.component';
+import { ProfileComponent } from './profile/profile.component';
 
 const routes: Routes = [
   {
     path: '',
     component: HomeComponent,
     canActivate: [AuthGuard],
   },
+  {
+    path: ':username',
```

Comment:
> Does this need a / in front of it, or does that get handled automatically?

A leading `/` isn't required for these routes.

> I wonder if it might make more sense to have this be loaded from the app-routing file instead? Because the profile page is a fully separate page from the home component, which will have the gists and top repos. Room for discussion and opinions on routing strategy here - the other Angular project almost feels a bit too complex, but this doesn't quite feel right here to me either. 🤔

We can probably simplify the routing here. I had originally done that but it made too many changes elsewhere and I wanted to avoid merge conflicts since that would be outside the scope of my patch.